frontend/app/app.js

```diff
@@ -0,0 +1,122 @@
+var app = angular.module('movies', []);
+var API = "http://localhost:8080/api/"; //Your server url goes here
+
+/**
+ * Here we define a new controller called MoviesController. 
+ * We require $http, so angular injects it automatically. 
+ */
+app.controller("MoviesController", ['$http', function($http) {
+
+	var collection = this; //Save this controller 
+	/**
+	 * Use the $http method to send a GET request to our movie API. 
+	 * We provide some details in the first paramater. 
+	 * We use the .then() method to pass in our success and error functions. 
```

Comment:
Might want to talk about how `$http` returns a promise and `.then` allows us to unwrap the promise.